schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -1752,42 +1801,44 @@
     "Spec.Distribution.Modules.Dr.Velero": {
       "type": "object",
       "additionalProperties": false,
+      "description": "Configuration for the Velero package.",
       "properties": {
         "retentionTime": {
           "type": "string",
-          "description": "The retention time for velero"
+          "description": "CURRENTLY NOT IS USE."
         },
         "backend": {
           "type": "string",
           "enum": [
             "minio",
             "externalEndpoint"
           ],
-          "description": "The backend for velero"
+          "description": "The storage backend type for Velero. `minio` will use an in-cluster MinIO deployment for object storage, `externalEndpoint` can be used to point to an external object storage instead of deploying an in-cluster MinIO."
```

Comment:
```suggestion
          "description": "The storage backend type for Velero. `minio` will use an in-cluster MinIO deployment for object storage, `externalEndpoint` can be used to point to an external S3-compatible object storage instead of deploying an in-cluster MinIO."
```